libraries/AP_PiccoloCAN/AP_PiccoloCAN.cpp

```diff
@@ -344,6 +344,14 @@ void AP_PiccoloCAN::update()
         if (is_esc_channel_active(ii)) {
 
             uint16_t output = 0;
+
+            if (SRV_Channels::channel_function(ii) == SRV_Channel::k_throttle) {
```

Comment:
This will conflict with the ECU mapping below: https://github.com/ArduPilot/ardupilot/pull/28466/files#diff-c699548ddb0422e17ef8a505c29a96e536c9a3867856f18b0d82290c907cd129R367